config/default.py

```diff
@@ -11,6 +11,8 @@
 POOL_SIZE = 4
 LOGGING_CONFIG = 'logging.conf'
 REQUEST_TIMEOUT_SECONDS = 3600  # 60 minutes
+DEFAULT_MAX_RUNTIME = 7200 # 2 hours
```

Comment:
it may be good to add comments for these variables to explain these a little better to the user, what's the relationship between these variables and the aggregation variables?

Reply:
Adding clarifying comments to config file